genealogos-frontend/index.html

```diff
@@ -0,0 +1,264 @@
+                            placeholder="http://localhost:8000/api">
+                        <small class="form-text text-muted">Enter the address of the Genealogos API. This is the
+                            endpoint where the flake ref, attribute path, and any other settings will be send for
+                            processing. For example:
+                            <ul>
+                                <li>https://genealogos.tweag.io/api</li>
+                                <li>https://127.0.0.1:8000/api</li>
+                            </ul>
+                        </small>
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-primary" onclick="saveSettings()">Save</button>
+                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz"
+        crossorigin="anonymous"></script>
+    <script>
+        // Global variables for flake ref and attribute path
+        let latestFlakeRef = "";
+        let latestAttributePath = "";
+        let apiAddress = "http://localhost:8000/api"; // Default API address
+
+        // Always set the copy and save button to disabled on page load, this fixes a firefox bug
+        document.querySelector(".btn-secondary:first-of-type").disabled = true;
+        document.querySelector(".btn-secondary:last-of-type").disabled = true;
+
+        function generateSBOM() {
+            // Get the flake ref and attribute path from the input fields
+            const flakeRef = document.getElementById("flake-ref").value;
+            const attributePath = document.getElementById("attribute-path").value;
+            const cycloneDxVer = document.getElementById("sbom-version").value;
+
+            document.getElementById("sbom-container").classList.remove("alert", "alert-warning"); // Remove error class if present
+
+            // Set the content of the sbom-container to a loading spinner
+            document.getElementById("sbom-container").innerHTML = `
+                <div class="text-center">
+                    <div class="spinner-border" role="status">
+                        <span class="visually-hidden">Loading...</span>
+                    </div>
+                </div>
+            `;
+            // Disable the generate SBOM button
+            document.querySelector(".btn-primary").disabled = true;
+
+            // Make a request to the API with flake_ref and attribute_path as get parameters
+            const response = fetch(`${apiAddress}/analyze?flake_ref=${flakeRef}&attribute_path=${attributePath}&cyclonedx_version=v${cycloneDxVer}`)
```

Comment:
I think nothing is using `response`, right? 
```suggestion
            fetch(`${apiAddress}/analyze?flake_ref=${flakeRef}&attribute_path=${attributePath}&cyclonedx_version=v${cycloneDxVer}`)
```

Reply:
(or return it, so that it can be chained with other promises)